source/PlayState.hx

```diff
@@ -723,9 +738,20 @@ class PlayState extends MusicBeatState
 					add(stageCurtains);
 			}
 		}
-		var gfVersion:String = 'gf';
 
-		switch (SONG.gfVersion)
+		//defaults if no gf was found in chart
+		var gfCheck:String = 'gf';
+		
+		if (SONG.gfVersion == null) {
+			switch(storyWeek)
+			{
+				case 4: gfCheck = 'gf-car';
+				case 5: gfCheck = 'gf-christmas';
+				case 6: gfCheck = 'gf-pixel';
+			}
+		} else {gfCheck = SONG.gfVersion;}
+
+		switch (gfCheck)

```

Comment:
This switch case is redundant because of the one you added. It also causes a building error due to the gfVersion variable no longer existing. I would recommend doing this instead.
```haxe
		//defaults if no gf was found in chart
		var gfCheck:String = 'gf';
		
		if (SONG.gfVersion == null) {
			switch(storyWeek)
			{
				case 4: gfCheck = 'gf-car';
				case 5: gfCheck = 'gf-christmas';
				case 6: gfCheck = 'gf-pixel';
			}
		} else {gfCheck = SONG.gfVersion;}

		gf = new Character(400, 130, gfCheck);
```
![2021-06-11_07-50-23_cmd](https://user-images.githubusercontent.com/15317421/121705679-bd49e900-ca89-11eb-81db-adff21aff747.png)

Reply:
oh, thats why I still have the Terminal open